examples/compose/docker-compose-multi-datacenter.yml

```diff
@@ -0,0 +1,37 @@
+version: '2.1'
```

Comment:
Personally, I don't see a good reason to run multi-DC on the same local host and would rather not have this file here. Unless you see a reason to iterate on multi-DC locally and that being something many users of this pattern might do.